src/script/assets/AssetMetaDataBuilder.ts

```diff
@@ -112,7 +112,7 @@ const buildMetadataVideo = (videoFile: File | Blob): Promise<Asset.VideoMetaData
  * This needs to be done because error events are not standardized between browser implementations.
  * @private
  * @param {Event} event - Error event
- * @returns {MediaError} Error object
```

Comment:
I think you can remove the jsdoc comment altogether. It doesn't add any value now :)
(same for the others)